### DISCUSSION TITLE
Sonoff basic r4 - basic switch

### DISCUSSION THREAD

**Discussion by palovan** [RavenSystem/esp-homekit-devices#2281] [MEPLHAA Scripts]
Hello everyone,

can you help me build MEPLHAA Script for Sonoff Basic R4 to run as basic switch? I can't make it work properly (relay is not switching) with example code: 

{
  "c": { 
    "io": [ [ [ 12, 13 ], 2 ], [ [ 0 ], 6 ] ],
    "l": 13, "b": [ [ 0, 5 ] ]
  },
  "a": [{
    "t": 1,
    "s": 5,
    "0": { "r": [ [ 12, 0 ] ] },
    "1": { "r": [ [ 12, 1 ] ] },
    "b": [ [ 0 ] ]
  }]
}

Thank you

**Comment by vglucas** [comment]
```json
{"c":{"io":[[[4,6],2],[[5],6,0,2,1,2000],[[9],6]],"c":1,"l":6,"b":[[9,5]]},"a":[{"0":{"r":[[4]]},"1":{"r":[[4,1]]},"b":[[5],[9]]}]}
```
